packages/hardhat-core/test/internal/hardhat-network/provider/node.ts

```diff
@@ -495,12 +511,29 @@ describe("HardhatNode", () => {
 
       it("mines a block with correct timestamp after time increase", async () => {
         const now = getCurrentTimestamp();
-        node.increaseTime(new BN(30));
+        const delta = 30;
+        node.increaseTime(new BN(delta));
+        await assertIncreaseTime(now + delta);
+      });
+
+      it("mining a block having increaseTime called twice counts both calls", async () => {
+        const now = getCurrentTimestamp();
+        const delta = 30;
+        node.increaseTime(new BN(delta));
+        node.increaseTime(new BN(delta));
         await node.mineBlock();
+        await assertIncreaseTime(now + delta * 2);
```

Comment:
`assertIncreaseTime` calls `mineBlock`, so I guess the previous line should be removed?

Reply:
Actually, I have a slight preference for removing the `mineBlock` call from the `assertIncreaseTime` and having to call it explicitly in the test.

Reply:
Ok, will do that

Reply:
Fixed